### ISSUE TITLE
Breadcrumb use nav element

### ISSUE THREAD

**Issue by markaplet** [Office-of-Digital-Innovation/California-State-Template#139]
Breadcrumb component is a small wayfinding navigation and should be wrapped with the nav landmark element and given an aria-label to identify it to AT users. Below is an example for recommendation:

```
<nav aria-label="You are here">
  <ol class="breadcrumb">
    <li class="breadcrumb-item">Home</li>
    <li class="breadcrumb-item active" aria-current="page">Current Page Name</li>
  </ol>
</nav>
```

Note: may sites are moving towards an accessible label such as "You are here" as an improvement to "Breadcrumb". The argument is that breadcrumb is an engineering or systems term rather than an accessible label. Personally I think both are fine. I do find more and more sites using the "You are here" label so it must be gaining traction and polarity. So maybe it's best to jump on that bandwagon.

**Comment by kkoryaka** [comment]
This is good point @markaplet, we will incorporate this in our next State Template revision.

**Comment by kkoryaka** [comment]
This issue was fixed in 5.6.1 release. Closing this issue
